File: level1/ex20.js
function solution(s) {
  // 주어진 조건에 맞게 단어를 바꾸는 함수
  function changeWord(word) {
    word = word.split('');

    for (let i = 0; i < word.length; i++) {
      if (i % 2) word[i] = word[i].toLowerCase();
      else word[i] = word[i].toUpperCase();
    }

    return word.join('');
  }

  // 공백을 기준으로 파싱하여 배열로 저장
  const strArr = s.split(" ");

  for (let i = 0; i < strArr.length; i++) {
    strArr[i] = changeWord(strArr[i]);
  }

  return strArr.join(" ");
}

File: level1/ex20.py
# 주어진 word의 짝수 인덱스는 대문자로,
# 홀수 인덱스는 소문자로 변환하는 함수
def convert(word):
    list = []
    for i in range(len(word)):
        converted = word[i].upper() if i % 2 == 0 else word[i].lower()
        list.append(converted)     
    return ''.join(list)
            

def solution(s):
    # 공백 기준으로 split하여 단어 배열 구하기
    words = s.split(' ')
    # 각 단어에 대하여 convert 함수 실행
    convertedWords = list(map(convert, words))
    return ' '.join(convertedWords)